### ISSUE TITLE
Use pyqt5 with scriptscanner

### ISSUE THREAD

**Issue by jayich** [CampbellGroup/common#201]
This is a good first step to switching from pyqt4 to pyqt5.

https://github.com/CampbellGroup/common/issues/155#issuecomment-328991181

Changes here should not be merged into master, but into branches working on https://github.com/CampbellGroup/common/issues/155#issuecomment-328991181

**Comment by aransfor** [comment]
branch created here https://github.com/CampbellGroup/common/tree/AMR/script_scanner_pyqt5

**Comment by aransfor** [comment]
@jayich @theoriginaljuice I ran into a very big issue immediately, PyQt5 does not support python 2.7 and in fact does not have an installer for python 2.7 so one would have to do a custom unsupported build which is what I suppose anaconda does, however if PyQt5 or anaconda decides to be incompatible completely with 2.7 (at least in there installers of which only anaconda works) well all be up a creek... Any suggestions here would be helpful but its seems to me upgrading to PyQt5 is blocked by upgrading to Python3

https://stackoverflow.com/questions/32080304/issues-with-building-pyqt5-on-ubuntu-14-04

**Comment by jayich** [comment]
@aransfor , okay, good to know.  Definitely sounds like we are blocked.
